src/docs/reference/core/Configuration/README.md

```diff
@@ -91,21 +94,46 @@ For example, the Email module allows SMTP configuration via the `SmtpSettings` c
 However, you can override the site settings from the `Startup` class like this (note that we use `PostConfigure` to override the site settings values but if the module doesn't use site settings you can just use `Configure`):
 
 ```
-public void ConfigureServices(IServiceCollection services)
+services
+    .AddOrchardCms()
+    .ConfigureServices(tenantServices =>
+        tenantServices.PostConfigure<SmtpSettings>(settings =>
+        {
+            settings.Port = 255;
+        }));
+
+// Or if you want to make use of IShellConfiguration as seen above:
+services
+    .AddOrchardCms()
+    .ConfigureServices((tenantServices, serviceProvider) =>
+    {
+        // Instead of IShellConfiguration you could fetch the configuration 
+        // values from an injected Configuration instance here too. While that 
+        // would also allow you to access standard ASP.NET Core configuration 
+        // keys it wouldn't have support for all the hierarchical sources 
+        // detailed above.
+        var shellConfiguration = serviceProvider.GetRequiredService<IShellConfiguration>();
+        var password = shellConfiguration.GetValue<string>("SmtpSettings:Password");
+
+        tenantServices.PostConfigure<SmtpSettings>(settings =>
+        {
+            settings.Password = password;
+        });
+    });
+```
+
+This will thus make the SMTP port use this configuration despite any other value defined from site settings. The second example's configuration value can come from e.g. an `appsettings.json` file like below:
```

Comment:
```suggestion
This will make the SMTP port use this configuration despite any other value defined in site settings. The second example's configuration value can come from e.g. an `appsettings.json` file like below:
```